docs/docs/networking.md

```diff
@@ -111,8 +137,9 @@ Now let's say you have 3 instances of App ACME, for example 10.0.0.2:31000, 10.0
         }
     }]
 ```
-This configuration will create a FQN of `acme.marathon.l4lb.thisdcos.directory` at port :10000 which will load balance all instances of the application.  It is important to note that Marathon itself doesn't do anything with this configuration it is metadata that Marathon manages for the application.  A client in DC/OS could open a connection to `acme.marathon.l4lb.thisdcos.directory` at port :10000, which would route to 10.0.0.2:31000 which will route to the `containerPort` of :8080.
+This configuration will create a FQN of `acme.marathon.l4lb.thisdcos.directory` at port `:10000`, which will load balance all instances of the application.
```

Comment:
`FQN` →`FQDN` 
please add a link and/or full text

Reply:
👍